source/elements/oneDAL/source/algorithms/algorithm_inputs.rst

```diff
@@ -0,0 +1,4 @@
+Algorithm inputs
+=========================
+
+This section will explains
```

Comment:
I know it's just a placeholder, but please make it grammatically correct :) it's either "explains" or "will explain"
Same for outputs, processing, descriptor, and some others as well.

Reply:
fixed across all other places